src/Maestro/Client/src/Generated/Models/BuildData.cs

```diff
@@ -26,7 +26,20 @@ public BuildData()
         /// <summary>
         /// Initializes a new instance of the BuildData class.
         /// </summary>
-        public BuildData(string repository, string commit, string buildNumber, string branch = default(string), IList<AssetData> assets = default(IList<AssetData>), IList<int?> dependencies = default(IList<int?>))
```

Comment:
did you make these changes manually?? The code for the maestro client is autogenerated, it should always match what is in the prod deployment. Don't change it manually please.

Reply:
I think you will need to wait for a prod deployment before updating the client side of this change.

Reply:
> did you make these changes manually?? The code for the maestro client is autogenerated, it should always match what is in the prod deployment. Don't change it manually please.

I didn't notice it was autogenerated :-(. I'll look into that. Thanks for checking.

Reply:
> I think you will need to wait for a prod deployment before updating the client side of this change.

I see. because the client generator uses the prod swagger.json

Reply:
You can generate against INT (see the readme.md I think).  That updated client can then be checked into INT.